app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -313,6 +355,14 @@ def ca_cert
 
     private
 
+    def os_name_mapping(host)
+      host.operatingsystem.name =~ /redhat|centos/i ? 'rhel': host.operatingsystem.name.downcase
+    end
```

Comment:
I apologize for resurrecting an old PR, but I am testing this for Satellite 6.2 and I noticed that OS name will only match in these two cases. I tend to set OS name to "RHEL", I wonder if I should file an enhancement ticket to also include `rhel|red hat` cases.